src/core/ddsc/tests/CMakeLists.txt

```diff
@@ -92,3 +93,16 @@ set_tests_properties(
 		ENVIRONMENT "${CUnit_ddsc_config_simple_udp_env}")
 
 configure_file("config_env.h.in" "config_env.h" @ONLY)
+
+add_executable(oneliner
+  "oneliner.c"
+  "test_oneliner.c"
+  "test_oneliner.h"
+  "test_util.c"
+  "test_util.h")
+target_include_directories(
+  oneliner PRIVATE
+  "$<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/src/include/>"
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../ddsc/src>"
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../ddsi/include>")
```

Comment:
This'll work. It would've been cleaner to use the property from the `ddsc` target though.

Reply:
Hmm, ok, yes, that would be cleaner. This not being the only place (according to a quick grep) fixing it here won't solve the problem of fixing it everywhere. It is probably better to do that in another PR.

And thanks for giving it a quick check!